packages/botonic-react/tests/webchat/webchat.addmessages.test.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { Webchat } from '../../src/webchat/webchat'
+import { Document, Image, Video, Audio } from '../../src/components'
+import { act } from 'react-test-renderer'
+import {
+  renderUseWebchatHook,
+  expectToHaveRoles,
+  expectNotToHaveRoles,
+} from '../helpers/test-utils'
+import { render, screen } from '@testing-library/react'
+import { ROLES } from '../../src/constants'
+
+describe('TEST: webchat addMessageComponent', () => {
+  window.HTMLElement.prototype.scrollTo = function () {}
+
+  it('When adding an Image message the webchat has StyledWebchat, MessageList, StyledMessage and ImageMssage', async () => {
```

Comment:
it('TEST: When adding an Image message the webchat has StyledWebchat, MessageList, StyledMessage and ImageMessage'. <-- Mssage --> Message
Can you please put 'TEST:' for every "it" test?